caches/path-mapped/src/main/java/org/commonjava/maven/galley/cache/pathmapped/core/FileBasedPhysicalStore.java

```diff
@@ -51,4 +51,11 @@ public InputStream getInputStream( String storageFile ) throws IOException
         return new FileInputStream( new File( baseDir, storageFile ) );
     }
 
+    @Override
+    public boolean delete( FileInfo fileInfo )
+    {
+        File f = new File( baseDir, fileInfo.getFileStorage() );
+        return f.delete();
```

Comment:
Or worth to use Files.deleteIfExists(..) ? As the official doc said: 

> Note that the Files class defines the delete method to throw an IOException when a file cannot be deleted. This is useful for error reporting and to diagnose why a file cannot be deleted.

Reply:
Sounds good. Updated.